enrollment-server-onboarding-provider-innovatrics/src/main/java/com/wultra/app/onboardingserver/provider/innovatrics/InnovatricsApiService.java

```diff
@@ -145,4 +160,236 @@ public void deleteSelfie(final String customerId, final OwnerId ownerId) throws
         }
     }
 
+    /**
+     * Create a new customer resource.
+     * @param ownerId owner identification.
+     * @return optional of CreateCustomerResponse with a customerId.
+     * @throws RemoteCommunicationException in case of 4xx or 5xx response status code.
+     */
+    public Optional<CreateCustomerResponse> createCustomer(final OwnerId ownerId) throws RemoteCommunicationException {
+        final String apiPath = "/api/v1/customers";
+
+        try {
+            logger.info("Creating customer, {}", ownerId);
+            logger.debug("Calling {}", apiPath);
+            final ResponseEntity<CreateCustomerResponse> response = restClient.post(apiPath, null, EMPTY_QUERY_PARAMS, EMPTY_ADDITIONAL_HEADERS, new ParameterizedTypeReference<>() {});
```

Comment:
If headers and parameters are empty, could you use just `post(String path, Object request, ParameterizedTypeReference<T> responseType)`?